app/assets/stylesheets/helpers.scss

```diff
@@ -4,18 +4,15 @@
 }
 
 .wide-background-room {
-  background: white;
-  margin: 0 -600rem;
-  padding: 0.25rem 600rem;
-  //TODO - samuel: need to find a solution for this
-  height: 302px;
+  margin-top: -299px;
```

Comment:
This is very annoying, can't figure out what that extra pixel is but I had to let it go